pkg/gatherers/clusterconfig/sap_vsystem_iptables_logs.go

```diff
@@ -29,41 +30,36 @@ import (
 //   * 4.6.25+
 //   * 4.7.5+
 //   * 4.8+
-func GatherSAPVsystemIptablesLogs(g *Gatherer, c chan<- gatherResult) {
-	defer close(c)
-
+func (g *Gatherer) GatherSAPVsystemIptablesLogs(ctx context.Context) ([]record.Record, []error) {
 	dynamicClient, err := dynamic.NewForConfig(g.gatherKubeConfig)
 	if err != nil {
-		c <- gatherResult{errors: []error{err}}
-		return
+		return nil, []error{err}
 	}
 
 	kubeClient, err := kubernetes.NewForConfig(g.gatherProtoKubeConfig)
 	if err != nil {
-		c <- gatherResult{nil, []error{err}}
-		return
+		return nil, []error{err}
 	}
 
-	datahubsList, err := dynamicClient.Resource(datahubGroupVersionResource).List(g.ctx, metav1.ListOptions{})
+	datahubsList, err := dynamicClient.Resource(datahubGroupVersionResource).List(ctx, metav1.ListOptions{})
 	if errors.IsNotFound(err) {
-		c <- gatherResult{nil, nil}
-		return
+		klog.Info("SAP resources weren't found")
```

Comment:
It would be probably better to mention the `datahubGroupVersionResource` in the info log, but we can do this anytime.